hack/doc/samples/kubernetes-all.md

```diff
@@ -0,0 +1,109 @@
+
+## Dynatrace Kubernetes Monitoring (ActiveGate) (ClusterRole)
+
+|Resources accessed |API group |APIs used |Resource names |
+|------------------ |--------- |--------- |-------------- |
+|Nodes |`core` |List/Watch/Get | |
```

Comment:
Should regenerate, code does not mention `core`

Reply:
Removed the samples completely.